data_diff/dbt.py

```diff
@@ -301,6 +306,18 @@ def _cloud_diff(diff_vars: TDiffVars, datasource_id: int, api: DatafoldAPI) -> N
         diff_percent_list = {
             x.column_name: str(x.match) + "%" for x in diff_results.values.columns_diff_stats if x.match != 100.0
         }
+        columns_added = diff_results.schema_.exclusive_columns[1]
+        columns_removed = diff_results.schema_.exclusive_columns[0]
+        column_type_changes = diff_results.schema_.column_type_differs
```

Comment:
Do we need to show reordered columns as well?

Reply:
I think we could tackle that in a separate issue if it comes up